docs/decisions/0004-cross-service-data-linking.rst

```diff
@@ -0,0 +1,69 @@
+0004 Cross-service data linking
+###############################
+
+Status
+******
+
+**Accepted** February 2023
+
+Context
+*******
+
+The enterprise-subsidy service is a central piece in the redemption and fulfillment flow
+that results in the creation of enterprise enrollments via Subsidy value. Notably, there are
+bi-directional references between the ``subsidy access policy`` app of the ``enterprise-access`` service,
+and the enrollments interface of ``edx-enterprise``.  These two-way references help ensure
+the veracity of our business records and provide a path for robust reporting on the history
+of a given subsidized enterprise course enrollment.
+
+Decision
+********
+
+There are several model fields in the ``enterprise-subsidy`` service, and its main plugin,
+the ``openedx-ledger`` library that achieve the desired bi-directional reference property.
+
+This `entity-relationship diagram`_ demonstrates the references.
+
+
+References with enterprise-access
+=================================
+
+The ``SubsidyAccessPolicy`` model's table contains a reference to the ``uuid`` of ``LearnerCreditSubsidy``
+(or any other implementation of the abstract ``Subsidy`` model).  This is necessary for the policy to
+know which subsidy record should have the transaction for a given redemption action.
+
+In ``enterprise-subsidy``, the Ledger ``Transaction`` table will contain a ``subsidy_access_policy`` field
+that refers to the access policy record that allowed a given transaction to be created.
```

Comment:
I think I accidentally called it `policy_uuid` in my PR

https://github.com/openedx/openedx-ledger/pull/3/files

I think `subsidy_access_policy` is more explicit, but what about `subsidy_access_policy_uuid`?

Reply:
we discussed and agreed on `subsidy_access_policy_uuid` over slack